src/renderers/native/ReactNative.js

```diff
@@ -21,13 +21,15 @@ var ReactUpdates = require('ReactUpdates');
 
 var findNodeHandle = require('findNodeHandle');
 
+import type { ReactElement } from 'ReactElementType';
+
 ReactNativeDefaultInjection.inject();
 
 var render = function(
-  element: ReactElement<any>,
+  element: ReactElement | string | number | null | false,
```

Comment:
https://facebook.github.io/react/docs/glossary.html#react-nodes

Looks like the docs says that ReactNode cannot be `false` and can be an array